modules/local/vep_annotation_download.nf

```diff
@@ -0,0 +1,91 @@
+process VEP_DOWNLOAD {
+    tag "vep"
+    label 'process_long'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bcftools:1.20--h8b25389_0':
+        'biocontainers/bcftools:1.20--h8b25389_0' }"
+
+    input:
+    val genome
+    val vep_cache_version
+
+    output:
+    path("vep_cache")      , emit: vep_cache
+    path("vep_plugin_files.csv")  , emit: plugin_file
+    path "versions.yml"    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def gnomad_version2download = "${genome}".contains("38") ? "4.0": "2.1.1"
```

Comment:
Not 4.1? :)

Reply:
Good point, but for the time being we are using 4.0, so I wanted to keep it like this, it can be modified later on :)